src/sentry/coreapi.py

```diff
@@ -382,24 +394,24 @@ def should_filter(self, project, data, ip_address=None):
         so that we can store it in metrics
         """
         if ip_address and not is_valid_ip(project, ip_address):
-            return (True, 'ip_address')
+            return (True, 'ip-address')
 
         release = data.get('release')
         if release and not is_valid_release(project, release):
-            return (True, 'release_version')
+            return (True, 'release-version')
 
         message_interface = data.get('sentry.interfaces.Message', {})
         error_message = message_interface.get('formatted', ''
                                               ) or message_interface.get('message', '')
         if error_message and not is_valid_error_message(project, error_message):
-            return (True, 'error_message')
+            return (True, 'error-message')
 
         for filter_cls in filters.all():
             filter_obj = filter_cls(project)
             if filter_obj.is_enabled() and filter_obj.test(data):
-                return (True, 'other_filter')
+                return (True, filter_obj.id)
```

Comment:
It'd be good if this was casted to a string for consistency.

Reply:
Is this solely being returned for the purposes of logging here? 
https://github.com/getsentry/sentry/blob/a4862a028efee45baaaf975bcba181bcb737a83d/src/sentry/web/api.py#L372-L373

Reply:
The fact that these IDs aren't included in the keys-to-values mapping but occupy the same variable seems a bit like a code smell to me (I wouldn't say it's a particularly intuitive behavior.)

Reply:
Logging and to be a part of the granular stats in the chart rather than just being grouped together as "other_filter". These filters include web crawlers, legacy browsers, localhost, etc.